src/debugAdapter/goDebug.ts

```diff
@@ -427,15 +427,14 @@ class Delve {
 					if (mode === 'debug') {
 						this.noDebug = true;
 						const runArgs = ['run'];
-						const runOptions: { [key: string]: any } = { env };
+						const runOptions: { [key: string]: any } = { cwd: dirname, env };
```

Comment:
@simpleapples, @hyangah I think I am missing something... I am not seeing a net change here.
Given that `dirname` is the same as `program` when `isProgramDirectory` is true, setting the `cwd` upfront here and the code before would essentially result in the same behavior

Of course, for simplicity sake, the change in this PR is good. I just want to confirm that this change doesnt change any behavior to the end user

Reply:
i reviewd the change again and fix some typo. i think it won't change any behavior to the end user.